backend/server/src/http/mod.rs

```diff
@@ -33,10 +34,15 @@ type Request<T = Body> = hyper::Request<T>;
 pub(crate) async fn serve(
     config: &Config,
     api_root: api::RootNode,
-    api_context: api::Context,
 ) -> Result<()> {
     let assets = Assets::init(&config.assets).await.context("failed to initialize assets")?;
-    let ctx = Arc::new(Context::new(api_root, api_context, assets));
+
+    let db = db::create_pool(&config.db).await
+        .context("failed to create database connection pool (database not running?)")?;
+    db::migrate(&mut *db.get().await?).await
+        .context("failed to check/run DB migrations")?;
```

Comment:
I'm not very thrilled about having this in `http::serve`. Seems wrong to me. I would keep them in `main.rs` and just pass the db pool to `serve`.

Reply:
Moved it back. :)